### ISSUE TITLE
Wallet does not work on mainnet_candidate testnet

### ISSUE THREAD

**Issue by piotr-iohk** [cardano-foundation/cardano-wallet#1904]
# Context
 
Both `shelley_testnet` and `mainnet_candidate` run on cardano-node 1.15.1.
Wallet seems to work fine on `shelley_testnet` fails to start on `mainnet_candidate`. 

One difference I noticed is that `mainnet_candidate` has `"minUTxOValue": 1000000` whereas `shelley_testnet` has `"minUTxOValue": 0` in genesis. Could be unrelated.

| Information  | -                                                     |
| ---          | ---                                                   |
| Version      | node 1.15.1, wallet `2020.7.6 (git revision: df141f5698d0502d9335ff9232f2eb02df37afb2)` |
| Platform     | Linux         |
| Installation | from source           |

# Steps to Reproduce
https://hydra.iohk.io/job/Cardano/iohk-nix/cardano-deployment/latest/download/1/index.html

1. start wallet + node on `mainnet_candidate`

## Expected behavior

wallet should sync.

## Actual behavior
 
wallet syncs to slot 302, epoch 0 only (block 293, sometimes 292 or 294). In the log there is:
```
[cardano-wallet.network:Info:31] [2020-07-14 10:55:58.08 UTC] Intersection found: 01010101
[cardano-wallet.pools-engine:Error:31] [2020-07-14 10:55:58.08 UTC] Unexpected error following the chain: ExceptionInLinkedThread ThreadId 32 (DeserialiseFailure 90 "Size mismatch when decoding Block.\nExpected 4, but found 2.")
[cardano-wallet.network:Notice:31] [2020-07-14 10:55:58.08 UTC] Destroying cursor connection at ThreadId 32
[cardano-wallet.pools-engine:Error:31] [2020-07-14 10:55:58.08 UTC] Chain follower exited with error. Worker will no longer monitor stake pools.

```

---

# Resolution 

---

# QA

**Comment by rvl** [comment]
This will be fixed by #1909.

**Comment by piotr-iohk** [comment]
yes.
